From 12ffb9240bbfc0427ee428d8f6998fd93142c57d Wed, 7 Jul 2021 14:22:45 +0200
From: Andrew Branson <andrew.branson@jolla.com>
Date: Wed, 7 Jul 2021 14:22:10 +0200
Subject: [PATCH] Completely disable doc generation because we don't have xsltproc either.

diff --git a/meson.build b/meson.build
index 443ccc3..f8ecc7f 100644
--- a/meson.build
+++ b/meson.build
@@ -38,8 +38,9 @@
 subdir('libgupnp')
 subdir('tests')
 subdir('tools')
-subdir('doc')
-
+if get_option('gtk_doc')
+  subdir('doc')
+endif
 if get_option('vapi') and get_option('introspection')
     subdir('vala')
 endif
